cinema/serializers.py

```diff
@@ -68,6 +77,7 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
```

Comment:
You need annotate method to your MovieSessionViewSet, to make this field available